src/api/spec/policies/project_policy_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe ProjectPolicy do
+  let(:anonymous_user) { create(:user_nobody) }
+  let(:user) { create(:confirmed_user) }
+  let(:other_user) { create(:confirmed_user) }
+  let(:admin_user) { create(:admin_user) }
+  let(:project) { user.home_project }
+  subject { ProjectPolicy }
+
+  permissions :create?, :can_create_package_in?, :unlock?, :local_project_and_allowed_to_create_package_in? do
+    it { expect(subject).to permit(user, project) }
+    it { expect(subject).not_to permit(anonymous_user, project) }
+    it { expect(subject).not_to permit(other_user, project) }
+    it { expect(subject).to permit(admin_user, project) }
+  end
+
+  permissions :local_project_and_allowed_to_create_package_in? do
+    it { expect(subject).not_to permit(user, '') }
```

Comment:
I think we should text this also for other kind of users :thinking:

Reply:
not necessary. There is no different behavior if it's a remote project and user is normal or user is admin. Unless you spot some specific use case. Otherwise I would ask you to approve it,

Reply:
ping @Ana06  could you please tell me if you see a different use case or please accept it? thank you.

Reply:
@vpereira 

>  @Ana06 could you please tell me if you see a different use case or please accept it? thank you.

My first comments were only about the Rubocop part, but I have other concerns unrelated to that. I'll add them later.

Reply:
@vpereira 

>  Otherwise I would ask you to approve it,

>  @Ana06 could you please tell me if you see a different use case or please accept it? thank you.

We don't need to hurry to approve it. I have some more concerns. I'll add them later.